src/Log.php

```diff
@@ -830,8 +830,8 @@ public static function getHistoryData(CommonDBTM $item, $start = 0, $limit = 0,
                             break;
 
                         default:
-                            $data["old_value"] = RichText::getTextFromHtml($item->getValueToDisplay($searchopt, $data["old_value"]), false, true);
-                            $data["new_value"] = RichText::getTextFromHtml($item->getValueToDisplay($searchopt, $data["new_value"]), false, true);
+                            $data["old_value"] = RichText::getTextFromHtml($item->getValueToDisplay($searchopt, $data["old_value"] ?? ''), false, true);
+                            $data["new_value"] = RichText::getTextFromHtml($item->getValueToDisplay($searchopt, $data["new_value"] ?? ''), false, true);
```

Comment:
W should probably not alter the value passed to ``$item->getValueToDisplay()``, but rather handle the fact that it may return null.
```suggestion
                            $data["old_value"] = RichText::getTextFromHtml($item->getValueToDisplay($searchopt, $data["old_value"]) ?? '', false, true);
                            $data["new_value"] = RichText::getTextFromHtml($item->getValueToDisplay($searchopt, $data["new_value"]) ?? '', false, true);
```

Reply:
Indeed, that was the initial intention